manifests/mod/passenger.pp

```diff
@@ -329,6 +315,12 @@
   Optional[String] $mod_package                                                              = undef,
   Optional[String] $mod_package_ensure                                                       = undef,
   Optional[String] $mod_path                                                                 = undef,
+  Optional[Integer] $passenger_max_request_queue_time                                        = undef,
```

Comment:
Adding new parameters in the same PR as removing deprecations is a bad practice IMHO. Makes it hard to follow what's going on.

Reply:
sure, will put another PR for new params.